src/operator/join.rs

```diff
@@ -256,6 +324,32 @@ where
 
         left.plus(&right)
     }
+
+    /// Incremental anti-join operator.
+    ///
+    /// Returns indexed Z-set consisting of the contents of `self`,
+    /// excluding keys that are not present in `other`.
+    pub fn antijoin<TS, I2>(&self, other: &Stream<Circuit<P>, I2>) -> Stream<Circuit<P>, I1>
+    where
+        TS: Timestamp + DeepSizeOf,
+        I1::Key: Default,
+        I1::Val: Default,
+        I1::Item: Default,
+        I1::Batcher: DeepSizeOf,
+        I2: IndexedZSet<Key = I1::Key, R = I1::R> + Send + DeepSizeOf,
+        I2::Val: DeepSizeOf + Clone + Ord + Hash,
+    {
+        let stream1 = self.shard();
+        let stream2 = other.distinct_incremental().shard();
+
+        stream1
+            .minus(
+                &stream1.join_generic::<TS, _, _, _, _>(&stream2, |k, v1, _v2| {
+                    std::iter::once((k.clone(), v1.clone()))
+                }),
```

Comment:
We can use a monotonic join here (`Builder`) since the output tuples will always be ordered

Reply:
This is not a stream join.  I don't think we have a monotonic incremental join, and it's not straightforward to build one because of how join works (queuing updates for future timestamps).